packages/core/src/renderer/getStyleSheetForBucket.ts

```diff
@@ -45,25 +45,31 @@ const styleBucketOrderingMap = styleBucketOrdering.reduce((acc, cur, j) => {
  */
 export function getStyleSheetForBucket(
   bucketName: StyleBucketName,
-  target: Document | undefined,
+  targetDocument: Document | undefined,
+  insertionPoint: HTMLElement | null,
   renderer: GriffelRenderer,
```

Comment:
nit: refactor this to use a DTO now? it would make things a bit clearer what's going on, particularly tests